script/example.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+import yaml
+import json
+from argparse import ArgumentParser
+
+def action_json(args):
+
+    with open('scripts/example_data.json') as json_file:
```

Comment:
The read operation and json handling could be more efficiently expressed with context managers to ensure proper file handling and avoiding manual file open/close.
```suggestion
    with open('scripts/example_data.json', 'r') as json_file:
        data = json.load(json_file)
        print(json.dumps(data, indent=2))
```